package.json

```diff
@@ -29,7 +29,7 @@
     "preversion": "npm test",
     "postversion": "git push origin master && git push --follow-tags",
     "pretest": "cross-env NODE_ENV=tsoa_test ts-node ./tests/prepare.ts",
-    "test": "cross-env NODE_ENV=tsoa_test mocha **/*.spec.ts --exit --compilers ts:ts-node/register",
+    "test": "tsc -p ./tests/tsconfig.json --noEmit && cross-env NODE_ENV=tsoa_test mocha **/*.spec.ts --exit --require ts-node/register",
```

Comment:
I discovered while adding new unit tests that the `./tests/` folder was not actually compiling. This went undetected because (a) we don't have husky running the tests (but we could if https://github.com/lukeautry/tsoa/pull/375 is merged) and (b) even if we did, ts-node kinda sorta bypasses the compilation. So I wanted to explicitly compile the tests directory to make sure that we don't have bugs creep in.